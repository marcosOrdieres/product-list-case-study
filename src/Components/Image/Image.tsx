import React from 'react';
import styled from 'styled-components';

const ImageStyled = styled.img`
  width: 100%;
  height: 100%;
  object-fit: cover;
`;

export interface ImageProps {
  src?: string,
};

const Image = (props: ImageProps) => {
  return (
    <ImageStyled src={props.src} />
  )
};

export default Image;