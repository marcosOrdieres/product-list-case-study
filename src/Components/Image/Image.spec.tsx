import React from 'react';
import { render } from '@testing-library/react';

import Image from './Image';

describe('Image', () => {
  it('should render Image and match styles', () => {
    const { asFragment } = render(
      <Image src='' />
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
