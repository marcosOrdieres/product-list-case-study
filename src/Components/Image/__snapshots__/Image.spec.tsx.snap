// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Image should render Image and match styles 1`] = `
<DocumentFragment>
  <img
    class="sc-bdfBQB gZdlyO"
    src=""
  />
</DocumentFragment>
`;
